package.json

```diff
@@ -34,6 +34,7 @@
     "@vue/cli-plugin-unit-jest": "^4.5.6",
     "@vue/cli-service": "~4.5.0",
     "@vue/eslint-config-prettier": "^6.0.0",
+    "@vue/runtime-dom": "^3.2.37",
```

Comment:
This is required by Volar for built in typing (even though we don't use it yet).